dts/bindings/clock/nuvoton,npcx-pcc.yaml

```diff
@@ -11,6 +11,13 @@ properties:
     reg:
         required: true
 
+    ram-pd-depth:
+        required: false
+        type: int
+        description: |
+          Valid bit-depth of ram block power-down(PD) controllers for better
+          power consumption.
```

Comment:
Need to document what this means in a few more details and what the valid values are

(or could use an enum as suggested above)

Reply:
Done.